nailgun/entity_mixins.py

```diff
@@ -530,18 +563,33 @@ def read(self, entity=None, attrs=None, ignore=()):
 
 
 class EntityCreateMixin(object):
-    """A mixin that provides the ability to create an entity.
+    """This mixin provides the ability to create an entity.
 
-    The methods provided by this mixin work together to create an entity. A
-    typical tree of method calls looks like this::
+    The methods provided by this class work together. The call tree looks like
+    this::
 
         create
         └── create_json
             └── create_raw
-                ├── create_missing
-                └── create_payload
-
-    Only :meth:`create_raw` communicates with the server.
+                ├── create_payload
+                └── create_missing
+
+    In short, here is what the methods do:
+
+    :meth:`create_missing`
+        Populate required fields with bogus values. This method is not called
```

Comment:
s/bogus/random? Bogus to me implies that the values may not make sense (which is true) but the values we provide comply with the expected type at least.